system/modules/core/controllers/FrontendIndex.php

```diff
@@ -56,7 +56,9 @@ public function __construct()
 	 */
 	public function run()
 	{
+		/** @var \PageModel $objPage */
```

Comment:
You should remove this as it prevents line 93 from correctly noticing the variable could be a `\Model\Collection`